packages/medusa/src/services/price-list.ts

```diff
@@ -239,7 +239,13 @@ class PriceListService extends BaseService {
     const priceListRepo = this.manager_.getCustomRepository(this.priceListRepo_)
 
     const query = this.buildQuery_(selector, config)
-    return await priceListRepo.find(query)
+
+    const groups = query.where.customer_groups
+    delete query.where.customer_groups
```

Comment:
`query.where.customer_groups = undefined` is more performent since we do not alter the object itself. It is just a global comment and not necessarily applicable. I let you choose

Reply:
Setting to undefined breaks the repository queries since there will be an entry in `query.where` for `customer_groups` for this to work we should combine it with the `removeNullish` util which removes undefined properties from an object. 

I think using delete is better but please point out mistakes in the test: https://www.measurethat.net/Benchmarks/Show/18677/0/delete-performance-v2